README.md

```diff
@@ -15,9 +15,9 @@
 - [Potential Pitfalls](#potential-pitfalls)
 - [Contributors](#contributors-✨)
 
-## Updating to Version 8
+## Angular 10 Compatibility
```

Comment:
We don't specify a peer dependency?

Reply:
We do specify. But users in previous issues told that warnings from npm and yarn are not really informative regarding that :man_shrugging: